contracts/WildcardSteward_v2.sol

```diff
@@ -467,16 +481,59 @@ contract WildcardSteward_v2 is Initializable {
         address tokenPatron = currentPatron[tokenId];
 
         if (state[tokenId] == StewardState.Owned) {
-            uint256 totalToPayBack = price[tokenId];
-            // NOTE: pay back the deposit only if it is the only token the patron owns.
-            if (
-                totalPatronOwnedTokenCost[tokenPatron] ==
-                price[tokenId].mul(patronageNumerator[tokenId])
-            ) {
-                totalToPayBack = totalToPayBack.add(deposit[tokenPatron]);
-                deposit[tokenPatron] = 0;
-            }
+            _buy(tokenId, tokenPatron, wildcardsPercentage);
```

Comment:
There are two main buy funnels. One for when the token is not foreclosed, this is much the same as normal.

The second buy funnel is for when the token is in the foreclosed state, which will function as the auction mechanism state essentially

Reply:
This all implemented and addressed in latest push